lib/util.rb

```diff
@@ -87,4 +87,12 @@ def self.safe_write_to_file(filename, content)
   def self.send_email(...)
     EmailRenderer.sendmail("/", ...)
   end
+
+  def self.populate_ipv4_txt
+    ips = Address.where { (family(cidr) =~ 4) & (routed_to_host_id !~ id) }
+      .map { NetAddr::IPv4Net.new(it.cidr.network, NetAddr::Mask32.new(16)).to_s }
+    ips.uniq!
+    ips.sort!
```

Comment:
This does an alphabetic sort, so `9.9.0.0` comes after `10.10.0.0`.  It may be better to sort by address:

```suggestion
      .map { NetAddr::IPv4Net.new(it.cidr.network, NetAddr::Mask32.new(16)) }
    ips.sort!(&:cmp)
    ips.map!(&:to_s)
    ips.uniq!
```